### ISSUE TITLE
[ig/wot] Web of Things Interest Group rechartering

### ISSUE THREAD

**Issue by ashimura** [w3c/strategy#440]
New charter proposal, reviewers please take note.

# Charter Review

[Draft Charter:](https://w3c.github.io/wot-charter-drafts/wot-ig-2023-draft.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing IG recharter
* [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2021%2F12%2Fwot-ig-2021.html&doc2=https%3A%2F%2Fw3c.github.io%2Fwot-charter-drafts%2Fwot-ig-2023-draft.html)

---
Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 
* [Web of Things Community Group](https://www.w3.org/groups/cg/wot)
* [Web of Thing Japanese Community Group](https://www.w3.org/groups/cg/wot-jp)

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
* [GitHub repo for the discussion](https://github.com/w3c/wot-charter-drafts)

Anything else we should think about as we review? 
* To finalize the rechartering procedure, the IG would like to get a 3-month extension.

**Comment by ashimura** [comment]
Advance notice; https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0063.html (Member-only)

**Comment by ruoxiran** [comment]
no comments or requests from APA.

**Comment by himorin** [comment]
- `performance` should be removed from HR in 4. Coordination (I suppose it is better to align with the latest template, although HR might not be applicable for most of work...)
- opening dl is missing right after `WoT-related W3C Groups` line
- bunch of html syntax error (on dt/dd) in `External Organizations` part
- might need to add one last line in the history table

**Comment by simoneonofri** [comment]
No comments or requests on the Security side

**Comment by himorin** [comment]
no comment or request from i18n
i18n would thank on mentioning about considerations for non-Latin script in Scope section
and welcome interaction with your colleagues for discussion related to i18n.

**Comment by ashimura** [comment]
Thanks a lot, @ruoxiran @himorin @simoneonofri !

@himorin , I've fixed the [draft Charter HTML](https://w3c.github.io/wot-charter-drafts/wot-ig-2023-draft.html) based on your advice, and now it's a valid HTML as the [validator results](https://validator.w3.org/nu/?doc=https%3A%2F%2Fw3c.github.io%2Fwot-charter-drafts%2Fwot-ig-2023-draft.html) shows.

Also I've added one line about the Charter extension for the  rechartering process like the [current extended Charter](https://www.w3.org/2021/12/wot-ig-2021.html).

@plehegar , is there any concern/comment from the Privacy viewpoint?

**Comment by plehegar** [comment]
PING abstained to review the charter, so it's fine to go ahead.

**Comment by ashimura** [comment]
(TiLT approval has been requested, with a 3-month extension)

**Comment by himorin** [comment]
In section 1., it is written as `The priority in this fourth charter period is to`, but per history table this charter seems to be fifth.

**Comment by siusin** [comment]
The introduction to PlugFest and Testing is much appreciated, please continue this excellent work. 

A few minor suggestions:
- in **Mission**, s/by collection and analysis of use cases and requirements/through collection and analysis of use cases and requirements
- in 4.2, please make sure [Decentralized Identifier Working Group](https://w3c.github.io/wot-charter-drafts/wot-ig-2023-draft.html) and [Spatial Data on the Web Working Group](https://www.w3.org/web-networks/) are using the correct URLs
- [Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/) needs to be updated as Positive Work Environment at W3C: Code of Conduct

**Comment by ashimura** [comment]
Thanks a lot, @himorin and @siusin !

I've just fixed the [draft Charter](https://w3c.github.io/wot-charter-drafts/wot-ig-2023-draft.html) accordingly.

Please note I used not Charter URLs but WG page URLs for DID WG and SDW WG, because we use group page URLs within section 4.2 generally.

**Comment by plehegar** [comment]
One minor comment:

[Web and Automotive Business & Working Groups](https://www.w3.org/auto/) - We stopped our activities in this space so this should be dropped from the charter. You could replace

**Comment by ashimura** [comment]
Thanks a lot, @plehegar !

Just updated the draft Charter. Please see: https://www.w3.org/2024/04/wot-ig-2024.html#w3c-coordination

**Comment by himorin** [comment]
history table should have additional one line to briefly describe this rechartering.

**Comment by svgeesus** [comment]
Looks good
